test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,71 @@
+./test_scripts/API/SetAudioStreamingIndicator/059_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_comm_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/060_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_media_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_non-media_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/062_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_navi_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/063_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_comm_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_missing_:
+./test_scripts/API/SetAudioStreamingIndicator/065_ATF_N_wrong_Json_wrong_serviceType_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_serviceType_missing_wrong_wrongtype
+./test_scripts/API/SetAudioStreamingIndicator/066_ATF_N_wrong_Json_wrong_frameinfo_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_frameinfo_missing_wrong_wrongtype
+./test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_wrong_Json_wrong_rpcType_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_rpcType_missing_wrong_wrongtype
+./test_scripts/API/SetAudioStreamingIndicator/068_ATF_N_wrong_Json_wrong_rpcFunctionId_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_rpcFunctionId_missing_wrong_wrongtype
+./test_scripts/API/SetAudioStreamingIndicator/069_ATF_N_wrong_Json_wrong_rpcCorrelationId_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_rpcCorrelationId_missing_wrong_wrongtype
+./test_scripts/API/SetAudioStreamingIndicator/070_ATF_N_wrong_Json_missing_payload_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_missing_payload
```

Comment:
@OVikhrov, the requirement is "[GeneralResultCodes] INVALID_DATA wrong json syntax"
You can find it in attached Specification_SetAudioStreamingIndicator_v1.docx or in linked reqs with type "changes"